ghost/core/core/server/services/members/members-api/controllers/RouterController.js

```
@@ -351,16 +351,21 @@ module.exports = class RouterController {
         if (!member && options.email) {
+            const tokenData = {
+                email: options.email,
+                attribution: {
+                    id: options.metadata.attribution_id ?? null,
+                    type: options.metadata.attribution_type ?? null,
+                    url: options.metadata.attribution_url ?? null
+                }
+            }
+            if (options.metadata.newsletters) {
+                tokenData.newsletters = await this._validateNewsletters(JSON.parse(options.metadata.newsletters));
+                delete options.metadata.newsletters;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)